cocos/renderer/backend/opengl/CommandBufferGL.cpp

```diff
@@ -522,6 +522,9 @@ void CommandBufferGL::setLineWidth(float lineWidth)
 {
     if(lineWidth > 0.0f)
         glLineWidth(lineWidth);
+    else
+        glLineWidth(1.0f);
+    
```

Comment:
it is strange, `1.f > 0.0f`, so will be set, why need to add this line

Reply:
Default line width of some commands are `0`, that make `glLineWidth` remain unchanged if `setLineWidth` is not called, which state is wrong.

Reply:
So this codes changed to
```c++
if (lineWidth >= 0.f)
    glLineWidth(lineWidth);
```
?

Reply:
The default width is 1:0 in OpengGL. Values less than or equal to 0:0 generate the
error INVALID_VALUE.

Reply:
Then it is the error of our codes to pass 0?